schema.md

```diff
@@ -117,3 +140,14 @@
     {\"0\":\"liveAgentPc\"}
     {\"0\":\"liveAgentPage_0\",\"1\":\"http://www.alamy.com/help/what-is-model-release-property-release.aspx\"}
     ```
+* __file_name:__ 
+	* __Type:__ string
+	* __Description:__ Concatenation of the crawl_id and the called file, format (crawl_id)_(called file name)
```

Comment:
The "file" here is actually the JSON file corresponding to a single `location` in the raw collected data.